test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/SendingRequestEventVerifier.cs

```diff
@@ -228,31 +230,31 @@ internal void VerifySendingRequest(object sender, SendingRequest2EventArgs args)
                 {
                     if (args.RequestMessage.GetType().Name != "InternalODataRequestMessage")
                     {
-                        var requestMessage = args.RequestMessage as HttpWebRequestMessage;
+                        var requestMessage = args.RequestMessage as HttpClientRequestMessage;
 
                         this.assert.IsNotNull(requestMessage, "RequestMessage should be of type HttpWebRequestMessage");
 
-                        var httpWebRequest = requestMessage.HttpWebRequest as HttpWebRequest;
-                        this.assert.IsNotNull(httpWebRequest, "RequestMessage.HttpWebRequest should be of type HttpWebRequest");
+                        var httpRequestMessage = requestMessage.HttpRequestMessage as HttpRequestMessage;
+                        this.assert.IsNotNull(httpRequestMessage, "RequestMessage.HttpWebRequest should be of type HttpWebRequest");

```

Comment:
Please update the assert message. They still refer to `HttpWebRequestMessage` and `HttpWebRequest`.